src/components/TextInput.tsx

```diff
@@ -83,6 +83,7 @@ export class CTextInput extends React.Component<Props, State> {
             inputStyle,
           ]}
           value={value}
+          selectionColor={Colors.contentSecondary}
```

Comment:
i'm going with a slightly softer color here for now, and will check with Kayla / update it following our working session later today

Reply:
update on this @MuckT after Kayla and I caught up we landed on contentSecondary because it's a little softer in light mode so i'll leave this here :)